src/assets/release-notes-1.20.0-alpha.1.json

```diff
@@ -760,59 +212,10 @@
-    "commit": "75b555241578ac60cbdef21e01604f2bba8d040d",
-    "text": "Do not add nodes labeled with kubernetes.azure.com/managed=false to backend pool of load balancer.",
-    "markdown": "Do not add nodes labeled with kubernetes.azure.com/managed=false to backend pool of load balancer. ([#93034](https://github.com/kubernetes/kubernetes/pull/93034), [@matthias50](https://github.com/matthias50)) [SIG Cloud Provider]",
-    "author": "matthias50",
-    "author_url": "https://github.com/matthias50",
-    "pr_url": "https://github.com/kubernetes/kubernetes/pull/93034",
-    "pr_number": 93034,
-    "areas": ["cloudprovider", "provider/azure"],
-    "kinds": ["bug"],
-    "sigs": ["cloud-provider"]
-  },
-  "93043": {
-    "commit": "0a7050f531f8fa4de19e8aa4feb274aee4d8340c",
-    "text": "fix: determine the correct ip config based on ip family",
-    "markdown": "Fix: determine the correct ip config based on ip family ([#93043](https://github.com/kubernetes/kubernetes/pull/93043), [@aramase](https://github.com/aramase)) [SIG Cloud Provider]",
-    "author": "aramase",
-    "author_url": "https://github.com/aramase",
-    "pr_url": "https://github.com/kubernetes/kubernetes/pull/93043",
-    "pr_number": 93043,
-    "areas": ["cloudprovider", "provider/azure"],
-    "kinds": ["bug"],
-    "sigs": ["cloud-provider"]
-  },
-  "93052": {
-    "commit": "58893f34432e68a53b7abefcb9b924b22f8a3867",
-    "text": "fix: initial delay in mounting azure disk \u0026 file",
-    "markdown": "Fix: initial delay in mounting azure disk \u0026 file ([#93052](https://github.com/kubernetes/kubernetes/pull/93052), [@andyzhangx](https://github.com/andyzhangx)) [SIG Cloud Provider and Storage]",
-    "author": "andyzhangx",
-    "author_url": "https://github.com/andyzhangx",
-    "pr_url": "https://github.com/kubernetes/kubernetes/pull/93052",
-    "pr_number": 93052,
-    "areas": ["provider/azure"],
-    "kinds": ["bug"],
-    "sigs": ["cloud-provider", "storage"],
-    "duplicate": true
-  },
   "93078": {
     "commit": "e643286c06dd68077efc1153cd12c0661c2efed6",
-    "text": "Status of v1beta1 CRDs without \"preserveUnknownFields:false\" will show violation \"spec.preserveUnknownFields: Invalid value: true: must be false\"",
-    "markdown": "Status of v1beta1 CRDs without \"preserveUnknownFields:false\" will show violation \"spec.preserveUnknownFields: Invalid value: true: must be false\" ([#93078](https://github.com/kubernetes/kubernetes/pull/93078), [@vareti](https://github.com/vareti)) [SIG API Machinery]",
+    "text": "The status of v1beta1 CRDs without \"preserveUnknownFields:false\" now shows a violation, \"spec.preserveUnknownFields: Invalid value: true: must be false\".",
+    "markdown": "The status of v1beta1 CRDs without \"preserveUnknownFields:false\" now shows a violation, \"spec.preserveUnknownFields: Invalid value: true: must be false\". ([#93078](https://github.com/kubernetes/kubernetes/pull/93078), [@vareti](https://github.com/vareti))",
```

Comment:
Notes mapping files are applied!
This edit from @celestehorgan is here:

https://github.com/kubernetes/sig-release/blob/4ef1061d54fa0b53fa64dd0a82d3149bd167fe20/releases/release-1.20/release-notes/maps/pr-93078-map.yaml#L1-L3

Reply:
Yay, that's pretty cool!